metadata/v1.2/0/0-move-stdlib.json

```diff
@@ -0,0 +1,7 @@
+{
+    "title": "Update aptos framework to cb4ba0a57c998c60cbab65af31a64875d2588ca5",
```

Comment:
Let's name the title `Update aptos framework to v1.2` and then leave the hash detail in the description?

Reply:
we can also add the release note link in the description https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.2.4